routing/index_graph_starter.cpp

```diff
@@ -189,6 +189,15 @@ void IndexGraphStarter::RedressRoute(vector<Joint::Id> const & route,
     Joint::Id const prevJoint = route[i];
     Joint::Id const nextJoint = route[i + 1];
 
+    // If a route goes through a loop-edge starting from a joint and finishing
```

Comment:
It could happen if there's a feature with a loop like that one: http://www.openstreetmap.org/way/45423745#map=19/55.71633/37.52047